Lecture3.md

```diff
@@ -137,6 +149,9 @@ run :-  write('Enter the first number: '), read(FirstNumber),
 
 ?- length([1,2,3, ahmad], X).
 
-?- append([1,2,[a, b], X).
```

Comment:
Hi @Rami-Sabbagh, there is a missing `]` here, It was meant to be:

`append([1,2],[a, b], X).`